framerail/src/routes/+error.svelte

```diff
@@ -214,14 +214,11 @@ as soon as we can figure out prettier support for it.
       </form>
     {/if}
   {/if}
-{:else if $page.error.view === "page_permissions"}
+{:else if $page.error.view === "permissions"}
   UNTRANSLATED:Lacks permissions for page
   {@html $page.error.compiled_html}
-{:else if $page.error.view === "site_missing"}
-  UNTRANSLATED:No such site
-  {@html $page.error.html}
 {:else}
-  UNTRANSLATED:Fallback error, something really went wrong
+  UNTRANSLATED:Fallback error, invalid view type from backend
```

Comment:
This case is the generic fallback, I think we can keep the message as is, since the error might not actually be invalid view type. It could also be things like unable to connect to backend.

Reply:
That is true, though I feel like "went really wrong" is kind of not the best message. Like you mentioned, there are a number of issues that can cause this.

Maybe `Fatal error: Unable to display view`?

Reply:
That could do, I guess when we localize it in the future we can reword it to be more verbose

Reply:
Absolutely, yeah